### ISSUE TITLE
Websockets no longer connecting with Firefox on v2.4.1 and up.

### ISSUE THREAD

**Issue by KingHorse2015** [traefik/traefik#7856]
### Do you want to request a *feature* or report a *bug*?
Report a Bug

Bug

### What did you do?
Connected Oven Media Server through Traefik. To get Sub zero latency streaming. With the firsts test I was on traefik v2.3.6 and got it working with all browsers. A few days later I decided to update all my docker containers. And Firefox stopped playing, when using debug it hangs on make a connection. So it doesn't through a error. Just keeps in getting connected stage. I dropped back to v2.3.6 and it started working again, then I tested v2.3.7 and that also worked, then I tested v2.4.0 that also worked. 

### What did you expect to see?
Firefox is able to connect to the websocket, does the handshake with the webRTC server, and starts playing the content.

### What did you see instead?
Firefox isn't able to connect to the websocket.
![image](https://user-images.githubusercontent.com/15171668/106742959-0a223c80-661e-11eb-8ca4-300cf54983dc.png)

### Output of `traefik version`: (_What version of Traefik are you using?_)
V.2.4.1 or V2.4.2
FireFox version: 85.0

```
(paste your output here)
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
[entryPoints]
  [entryPoints.web]
    address = ":80"
    [entryPoints.web.http.redirections.entryPoint]
      to = "websecure"
      scheme = "https"

  [entryPoints.websecure]
    address = ":443"

[api]
  dashboard = true

[certificatesResolvers.lets-encrypt.acme]
  email = "n.vxxxxxx@gmail.com"
  storage = "acme.json"
  [certificatesResolvers.lets-encrypt.acme.tlsChallenge]

[providers.docker]
  watch = true
  network = "proxynet"

[providers.file]
  filename = "traefik_dynamic.toml"

```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
(paste your output here)
```

**Comment by KingHorse2015** [comment]
Totally missed that I was on v2.4.1, thought that I was on v2.4.2. Never checked. Seems to be fixed in v2.4.2.
Closing this one.

**Comment by KingHorse2015** [comment]
Duplicate of #7853
